ts/src/test/test.ts

```diff
@@ -298,20 +298,27 @@ export default class testMainClass extends baseMainTestClass {
                 if (tempFailure) {
                     // wait and retry again
                     await exchange.sleep (i * 1000); // increase wait seconds on every retry
+                    // if last retry was gone with same `tempFailure` error, then let's eventually return false
+                    if (i === maxRetries - 1) {
+                        dump ('[TEST_WARNING]', 'Method could not be tested due to a repeated Network/Availability issues', ' | ', exchange.id, methodName, argsStringified);
```

Comment:
as we agreed, even in case of repeated failures for those connectivity issues, we don't throw `[TEST_FAILURE]` special flag (so, `run-tests` will not mark this exchange tests as failed)